Just.xcodeproj/xcshareddata/xcschemes/Just.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0900"
```

Comment:
This change is due to Xcode 9.1, right? Let's get rid of it as well.

Reply:
Done.